src/diffusers/loaders.py

```diff
@@ -32,11 +32,11 @@
     LoRAAttnAddedKVProcessor,
     LoRAAttnProcessor,
     LoRAAttnProcessor2_0,
-    LoRALinearLayer,
     LoRAXFormersAttnProcessor,
     SlicedAttnAddedKVProcessor,
     XFormersAttnProcessor,
 )
+from .models.lora import Conv2dWithLoRA, LinearWithLoRA, LoRAConv2dLayer, LoRALinearLayer
```

Comment:
`Conv2dWithLoRA, LinearWithLoRA, LoRAConv2dLayer, LoRALinearLayer`

From these names, one would think `*WithLoRA` and LoRA*` are essentially the same. Could we distinguish them better?